pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -545,6 +545,36 @@ func TestZeroHintMath(t *testing.T) {
 				}),
 			},
 		},
+		"SignedPow": {
+			// Zero value - assertion failed, any other - nothing.
+			{
+				operanders: []*hintOperander{
+					{Name: "base", Kind: apRelative, Value: feltInt64(0)},
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newSignedPowHint(ctx.operanders["base"])
+				},
+				errCheck: errorTextContains("Cannot raise 0 to a negative power"),
```

Comment:
can you put the entire error text here?

Reply:
the full text is this : "assertion `Cannot raise 0 to a negative power` failed". I think the current impl is just fine. Though I can change if think that would be better. I was just following the design of `newSplitIntHint`.